src/mame/drivers/cyclemb.cpp

```diff
@@ -476,6 +520,7 @@ uint8_t cyclemb_state::skydest_i8741_0_r(offs_t offset)
 						m_mcu[0].packet_type^=0x20;
 						if(m_mcu[0].packet_type & 0x20)
 						{
+							m_dial_status[0].update = true;
```

Comment:
Is it needed? The only time this port is read is right here

Reply:
For some reason, dial_r is called one extra time, and without it, the delta cannot be calculated properly.
If I can find where called from can be removed, but I don't know so far, and looking into it now. Hmm...

Reply:
This was because also ioportmanager::frame_update() (ioport.cpp:2014) called ioport read().
Is there a better way than the current implementation?

Reply:
Ahh I see. No I don't have a good solution. For now, put back an update_dial function like you did before, sorry.
Can you also add to the TODO list?: "- separate driver into 2 classes"

Reply:
Okay, update_dial() is coming back, and adding TODO is done.

Reply:
Thanks